src/utils/eligibilityUtils.js

```
@@ -0,0 +1,45 @@
+    case "in":
+      return Array.isArray(conditionValues) && conditionValues.includes(userValue);
+    case "greater than equals":
+    case "greater_than_equals":
+      return Number(userValue) >= Number(conditionValues);
+    case "less than equals":
+    case "less_than_equals":
+      return Number(userValue) <= Number(conditionValues);
+    case "between":
+      if (!Array.isArray(conditionValues) || conditionValues.length !== 2)
+        throw new Error("Between condition requires an array of two values");
+      const [min, max] = conditionValues.map(Number);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)